memory-game-project/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.s[ac]ss$/i,
```

Comment:
one more reccomendation: try to use webpack-dev-server. It is very comfortable for developing because it compile code automatically and update the page in browser. 
https://webpack.js.org/guides/development/#using-source-maps
https://webpack.js.org/guides/development/#using-webpack-dev-server